server/routes/appointments/search/handlers/select-date.test.ts

```diff
@@ -0,0 +1,75 @@
+import { Request, Response } from 'express'
+import { plainToInstance } from 'class-transformer'
+import { validate } from 'class-validator'
+import SelectDateRoutes, { SelectDate } from './select-date'
+import { associateErrorsWithProperty } from '../../../../utils/utils'
+import { ServiceUser } from '../../../../@types/express'
+
+jest.mock('../../../../services/activitiesService')
+
+describe('Route Handlers - Appointments Management - Search Results', () => {
```

Comment:
Small copy paste issue. Presumably should be Route Handlers - Appointments Management - Search Select Date or similar